### ISSUE TITLE
Broken on 0.26

### ISSUE THREAD

**Issue by ssomnoremac** [johanneslumpe/react-native-gesture-recognizers#7]
Because of [this](http://stackoverflow.com/questions/37350035/rnmk-super-expression-must-either-be-null-or-a-function/37415044#37415044) of course but going in to change `React` and `Component` to import from `react` isn't fixing it. I'm not sure of the use for `_interopRequireDefault` but it seems to be culprit as my error is pointing to `babel-helpers`

**Comment by johanneslumpe** [comment]
The project uses babel 5 whereas RN now uses babel 6 - that might be part of the issue you're seeing?

**Comment by KalebPortillo** [comment]
How to fix the babel issue?

**Comment by ssomnoremac** [comment]
just get replace all the `_react-native2` references with `React` and do the things mentioned in the link I posted in the first comment:

```
 import React, { Component, PropTypes } from 'react';
 import { View } from 'react-native';`  
```

that's what got me to issue #9

**Comment by darmie** [comment]
I am still having an issue with Babel.

**Comment by funkjunky** [comment]
me too... 

`ReferenceError: [BABEL] /Users/funk/Development/Projects/ReactZeroNotifications/node_modules/react-native-gesture-recognizers/lib/index.js: Using removed Babel 5 option: /Users/funk/Development/Projects/ReactZeroNotifications/node_modules/react-native-gesture-recognizers/.babelrc.stage - Check out the corresponding stage-x presets http://babeljs.io/docs/plugins/#presets`

**Comment by cherniv** [comment]
Guys, did someone solve it?

**Comment by uriklar** [comment]
+1

**Comment by sidevesh** [comment]
+1

**Comment by AdamKraif** [comment]
+1

**Comment by johanneslumpe** [comment]
This should be fixed with ed416620eaeb9f35a60d68d4b876b4f41f349d1f
